db/data_migration/20171011123033_reslug_ukti_to_dit.rb

```diff
@@ -0,0 +1,84 @@
+require "data_hygiene/organisation_reslugger"
```

Comment:
I don't think we need to require this. The environment gets included when these get run https://github.com/alphagov/whitehall/blob/master/lib/tasks/data_migrator.rake#L6

Reply:
Ah right, was copying from prior art!